release-config.json

```
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "packages": {
+    ".": {
+        "release-type": "simple",
+        "include-v-in-tag": false,
+        "draft": false,
+        "pull-request-header": ":robot: Next release",
+        "pull-request-footer": " "
```

The first time it's added it helps to give it the starting commit

```suggestion
        "pull-request-footer": " "
        "bootstrap-sha": "ea8827dee679cd97cb7dd8b532063d32e36f89aa"
```